Stripe/PublicHeaders/STPSourceParams.h

```diff
@@ -49,25 +49,25 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  *  A set of key/value pairs that you can attach to a source object.
  */
-@property (nonatomic, copy, nullable) NSDictionary *metadata;
+@property (nonatomic, copy, nullable) NSDictionary<NSString *,NSString *>*metadata;
 
 /**
  *  Information about the owner of the payment instrument. May be used or required
  *  by particular source types.
  */
-@property (nonatomic, copy, nullable) NSDictionary *owner;
+@property (nonatomic, copy, nullable) NSDictionary<NSString *, id>*owner;
 
 /**
  *  Parameters required for the redirect flow. Required if the source is authenticated by 
  *  a redirect (`flow` is "redirect").
  */
-@property (nonatomic, copy, nullable) NSDictionary *redirect;
+@property (nonatomic, copy, nullable) NSDictionary<NSString *, id>*redirect;
```

Comment:
Why id instead of NSString for the values? Are there non-string types we need to support? I don't think our actual networking logic will work for any non-strings currently.